### ISSUE TITLE
Pin multiple timelines/lists and entirely unpin?

### ISSUE THREAD

**Issue by Fidgetcetera** [nolanlawson/pinafore#88]
I hope this seems fairly easy to understand - I'd like to be able to pin multiple lists via the "community" menu along the top bar, or choose to have absolutely nothing pinned in that spot.

**Comment by pwFoo** [comment]
Would be nice to move "Community" tab to settings and use the space to pin multiple timelines (locale + federated)? And maybe sortable in settings? Would be great!

I would like top nav:
````
Home | Local | Federated | Notifications | Search | Settings
```

**Comment by rikmendes** [comment]
+1000 on this one, I wish I could have more than one pinned item..

**Comment by jdaviescoates** [comment]
> Would be nice to move "Community" tab to settings

YES! I'm often forgetting where it is and go looking for it in setting and then get confused when I can't find it.

Then when I finally click on everything I realise Community is where I can do it, and then I want to be able to pin more than one.

**Comment by jet082** [comment]
I would love to be able to pin more than one thing in general.
